src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -28,6 +32,7 @@
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static final String LOGO_URL = "/images/InsurelyticsLogo.png";
```

Comment:
Seems like the logo doesn't appear on the right panel when I run it.

Reply:
Can u chk if u have the file in the local machine? 
main/src/main/resources/images/InsurelyticsLogo.png

Reply:
I think I do. Does anyone else have this problem? @chaitanyabaranwal @ybchen97 @olivercheok20

Reply:
works on @ybchen97 tho

Reply:
Works for me as well